src/docs/development/accessibility-and-localization/internationalization.md

```diff
@@ -96,6 +99,42 @@ More information about these app properties, the types they
 depend on, and how internationalized Flutter apps are typically
 structured, can be found below.
 
+<a name="advanced-locale"></a>
+## Advanced locale definition
+
+Some languages with multiple variants require more than just a language code to
+properly differentiate.
+
+For example, fully differentiating all variants of Chinese requires specifying
+the language code, script code, and country code. This is due to the existence
+of simplified and traditional script, as well as regional differences in the way
+characters are written within the same script type.
+
+In order to fully express every variant of Chinese for the country codes `CN`,
+`TW`, and `HK`, the list of supported locales should include:
+
+{% prettify dart %}
+// Full Chinese support for CN, TW, and HK
+supportedLocales: [
+  const Locale.fromSubtags(languageCode: 'zh'), // generic Chinese 'zh'
+  const Locale.fromSubtags(languageCode: 'zh', scriptCode: 'Hans'), // generic simplified Chinese 'zh_Hans'
+  const Locale.fromSubtags(languageCode: 'zh', scriptCode: 'Hant'), // generic traditional Chinese 'zh_Hant'
+  const Locale.fromSubtags(languageCode: 'zh', scriptCode: 'Hans', countryCode: ), // 'zh_Hans_CN'
+  const Locale.fromSubtags(languageCode: 'zh', scriptCode: 'Hant', countryCode: ), // 'zh_Hant_TW'
+  const Locale.fromSubtags(languageCode: 'zh', scriptCode: 'Hant', countryCode: ), // 'zh_Hant_HK'
+],
+{% endprettify %}
+
+This explicit full definition will ensure your app can distinguish between and provide
+the fully nuanced localized content to all combinations of these country codes. If a
```

Comment:
Will the app also benefit from script-code-specific localized content provided by the Material and Cupertino libraries? In other words, should apps do this as a matter of course, even if they're not providing any localizations of their own?

Reply:
Yes, if the script-code is specified (off the top of my head I recall for Chinese and Serbian), we will provide script-specific content. However, since different countries/regions express certain characters slightly differently, these nuances are lost without the country code.

Generic traditional chinese is currently using zh_TW, due to matching platforms the best.